sql/core/src/main/scala/org/apache/spark/sql/Dataset.scala

```diff
@@ -1805,9 +1805,6 @@ class Dataset[T] private[sql](
    * @since 1.6.0
    */
   def sample(withReplacement: Boolean, fraction: Double, seed: Long): Dataset[T] = {
-    require(fraction >= 0,
```

Comment:
Right after the deleted validation, the logical operator "Sample" is called. This one is duplicated.